pkg/sql/schema_changer_test.go

```diff
@@ -159,7 +159,7 @@ CREATE TABLE t.test (k CHAR PRIMARY KEY, v CHAR);
 
 func validExpirationTime(expirationTime int64) bool {
 	now := timeutil.Now()
-	return expirationTime > now.Add(sql.SchemaChangeLeaseDuration/2).UnixNano() && expirationTime < now.Add(sql.SchemaChangeLeaseDuration*3/2).UnixNano()
+	return expirationTime > now.Add(sql.SchemaChangeLeaseDuration / 2).UnixNano() && expirationTime < now.Add(sql.SchemaChangeLeaseDuration * 3 / 2).UnixNano()
```

Comment:
I hate the lack of spacing too, but `gofmt`'s word is gospel. I'm afraid you'll have to put this back the way it was for the "lint" build to succeed.

Reply:
No problem. Accidentally formatted with the wrong linter before commit.